README.md

```diff
@@ -65,6 +65,41 @@ One.sendInteraction("/interactionPath", { 'key' : 'value' });
 ```
 * See example of usage [here](https://github.com/thunderheadone/one-sdk-flutter/tree/master/examples/flutter_example/lib/main.dart#L128)
```

Comment:
Wrong link. Should be line 132.

Reply:
There is one in main.dart
See example of usage [here](https://github.com/thunderheadone/one-sdk-flutter/tree/master/examples/flutter_example/lib/main.dart#L65)